src/main/scala/Week1/ReverseCypher.scala

```diff
@@ -1,7 +1,11 @@
 package Week1
 object ReverseCypher extends App{
 
-    "foobar".foreach( ch => print(reverseChar(ch)))
+  def cypherReverse(input:String):String= {
+    var output = ""
+    input.foreach(ch => output+=(reverseChar(ch)))
```

Comment:
You should be able to just .map over the input and reverse the character where lastly you would return the input which would be a reversed string